maths/number_of_digits.rb

```diff
@@ -0,0 +1,24 @@
+# Given a number, find number of digits in it.
+
+def count_digits(n)
+    count = 0
+    temp = n
+    
+    if (n == 0)
+        return count + 1
```

Comment:
if count is always zero, then this should return 1.

```suggestion
        return 1
```